lib/linguist/languages.yml

```diff
@@ -4110,6 +4110,8 @@ Shell:
   - ".command"
   - ".fcgi"
   - ".ksh"
+  - ".mksh"
+  - ".pdksh"
```

Comment:
Neither pdksh used to nor mksh uses the shell name as script extension, we use just `.sh` or no extension in the general case, `.ksh` is seen rarely and more for AT&T ksh scripts.